src/types/basic-query-filter-from-test.js

```diff
@@ -124,9 +124,11 @@ QUnit.test("returns undefined against incompatible set", function(assert) {
     var fromQuery = new BasicQuery({
         filter: new BasicQuery.KeysAnd({ type: 'critical' })
     });
+		var res;
```

Comment:
Seems like the indenting is off here. Also the next line could be `try {` (with a space) while you're fixing this up elsewhere.